test/embedded/avr/testStdlibFunctioning.swift

```diff
@@ -0,0 +1,11 @@
+// RUN: %swift-frontend -emit-ir %s -target avr-none-none-elf -disable-objc-interop \
```

Comment:
Do we need `-disable-objc-interop` and `-disable-stack-protector`.

Reply:
not really... I'll remove them